src/middlewared/middlewared/plugins/smart.py

```diff
@@ -153,6 +153,12 @@ class Config:
             id
         )
 
+        await self.middleware.call(
```

Comment:
Should this be a _service_change? If smartd fails to reload I suppose we would like to know?